<launch>
<!-- Launch file for Livox AVIA LiDAR -->

  <arg name="rviz" default="true" />
  <rosparam command="load" file="$(find std_detector)/config/config_lixel.yaml" />
  <node 
		pkg="std_detector" 
		type="lixel_demo_cloud_pair" 
		name="lixel_demo_cloud_pair" 
		output="screen" 
	/>
  <param name="lidar_path" type="string" value = "/mnt/e/dataset/forSTD/log/737/"/>
  <param name="source_name" type="string" value = "737_Cur.pcd"/>
   <param name="target_name" type="string" value = "621_Loop.pcd"/>


  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find std_detector)/rviz_cfg/demo_lixel.rviz" />
  </group>

</launch>